circle.yml

```diff
@@ -39,3 +39,23 @@ deployment:
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
       - build/push-aws.sh
+  osx:
+    branch: master
+    commands:
+      - |
+          if [ -n "${BUILD_OSX-}" ]; then
+            set -eux
+            # There must certainly be a better way to move ~/cockroach
```

Comment:
I think moving the directory is fine; I'm just surprised we would do it here instead of at `checkout: post:` above.